ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
True, there's a bit of a contradiction there. I think we definitely want to be using sqlite for running the tests as long as we can because I'm encouraging folks to use a pre-commit git hook which will run the tests before every commit. Making those tests run as fast as possible is key in that scenario. I guess I'm agnostic about whether or not to use postgres for development of this tool. Mostly I swtiched over just to prove out the vagrant box I set locally.

I'll fix the database settings.